src/website/views/components/footer.pug

```diff
@@ -0,0 +1,415 @@
+            <div class="o-footer__end">
+              <ul class="a-menu">
+                <li>
+                  <a href="#">Link</a>
+                  </li>
+                <li>
+                  <a href="#">Link</a>
+                </li>
+                <li>
+                  <a href="#">Link</a>
+                </li>
+                <li>
+                  <a href="#">Link</a>
+                </li>
+              </ul>
+            </div>
+          </div>
+        </div>
+      </footer>
+  #a-tabs-panel--menu-inverse.a-tabs-panel
+    .-p--3(style="background-color:#eee;border-radius:.15em .15em 0 0;")
+      footer.o-footer.-inverse
+        .o-footer__content
+          .o-footer__footer
+            .o-footer__start
+              span © 2019, CenturyLink. All Rights Reserved.
+            .o-footer__end
+              ul.a-menu
+                li
+                  a(href="#") Link
+                li
+                  a(href="#") Link
+                li
+                  a(href="#") Link
+                li
+                  a(href="#") Link
+    :code(lang="html")
+      <footer class="o-footer -inverse">
+        <div class="o-footer__content">
+          <div class="o-footer__footer">
+            <div class="o-footer__start">
+              <span>© 2019, CenturyLink. All Rights Reserved.</span>
+            </div>
+            <div class="o-footer__end">
+              <ul class="a-menu">
+                <li>
+                  <a href="#">Link</a>
+                  </li>
+                <li>
+                  <a href="#">Link</a>
+                </li>
+                <li>
+                  <a href="#">Link</a>
+                </li>
+                <li>
+                  <a href="#">Link</a>
+                </li>
+              </ul>
+            </div>
+          </div>
+        </div>
+      </footer>
+
+h4 With subfooter
+ul#a-tabs--subfooter.a-tabs
+  li.-active
+    a(href='#a-tabs-panel--subfooter-base') Base
+  li
+    a(href='#a-tabs-panel--subfooter-inverse') Inverse
+.example.-mb--3
+  #a-tabs-panel--subfooter-base.a-tabs-panel.-active
+    .-p--3(style="background-color:#eee;border-radius:.15em .15em 0 0;")
+      footer.o-footer(style="background-color:#fff")
+        .o-footer__content
+          .o-footer__header
+            .o-footer__column
+              .-title Title example 1
+              a link 1
+              a link 2
+              a link 3
+              a link 4
+            .o-footer__column
+              .-title Title example 2
+              a link 1
+              a link 2
+              a link 3
+              a link 4
+            .o-footer__column
+              .-title Title example 3
+              a link 1
+              a link 2
+              a link 3
+              a link 4
+            .o-footer__column
+              .-title Title example 4
+              a link 1
+              a link 2
+              a link 3
+              a link 4
+        .a-divider
```

Comment:
@davidojedacl can this be achieved in other way? Including the divider and the footer bottom part in the footer content?